guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -8,4 +8,30 @@
 == Upgrade Warnings
 
 [id="katello-deprecations"]
-== Deprecations
+== Removals
```

Comment:
We always use the same headers (and remove them if they're empty). Please stick to them.

Reply:
@ekohl So here is the header incorrect? It should be Depreciations?

Reply:
No, depreciation is a monetary thing. Quoting https://www.baeldung.com/cs/deprecated-vs-obsolete

> Depreciation is a term used to refer to the decrease in the value of an item or thing, such as a car. In programming, it is often used mistakenly to refer to deprecated.

Reply:
@jeremylenz recommended we label this as a Removal, in theforeman repo currently has Depreciations in it so I wasn't sure if that gets kept.
[foreman-documentation/blob/3.8/guides/doc-Release_Notes/topics/katello.adoc](https://github.com/theforeman/foreman-documentation/blob/3.8/guides/doc-Release_Notes/topics/katello.adoc?plain=1)

Reply:
I've pushed an update and removed empty headers